src/Microsoft.AspNet.Mvc.WebApiCompatShim/ApiController.cs

```diff
@@ -93,16 +91,6 @@ public IPrincipal User
         }
 
         /// <summary>
-        /// Creates a <see cref="BadRequestResult"/> (400 Bad Request).
-        /// </summary>
-        /// <returns>A <see cref="BadRequestResult"/>.</returns>
-        [NonAction]
-        public virtual BadRequestResult BadRequest()
-        {
-            return new BadRequestResult();
-        }
```

Comment:
These methods should still be here and return the new result types. ApiController does not extend Controller